### DISCUSSION TITLE
ok.csv and ok.json examples don't work

### DISCUSSION THREAD

**Discussion by jpye-finch** [ghostfolio/ghostfolio#2647] [Q&A]
Unable to get .csv or .json to import. Partly due to this issue outlined here (need support for GBP) https://github.com/ghostfolio/ghostfolio/discussions/2317

Regardless your ok.csv and ok.json examples also don't import for me. I get the error "activities.0.currency ("USD") does not match with "undefined""
https://github.com/ghostfolio/ghostfolio/blob/main/test/import/ok.csv

Thanks.

**Comment by bats77** [comment]
I guess this section is still underwork, here's what I learned about structure of import csv : 
Type : 	Buy	Fee	Dividend	Interest	Liability	Sell	Valuable
Account	: (what you've defined in settings)				
? Update Cash Balance	true/false - doesn't seem working yet but shall be automatically accepted					
Name, Symbol, Isin	: (better to settle before, to have a code like RIO.L)					
Date :	DD/MM/YYYY						
Quantity : (not sure if it's accept many decimals)				
Unit Price	: (in US/EN standard with "." to separate cents, not sure how thousands are handled)				
Fee :	 	(US/EN standard)			
Note :							
Tags :			
Currency : (better settling them before importing)
